test/test_html_linter.py

```diff
@@ -570,6 +571,49 @@ def test_valid(self):
         self.assertEquals('', html_linter.lint(self.valid_html))
 
 
+class TestHTML5LinterScript(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.script = os.path.join(os.path.dirname(__file__),
+                                  '..',
+                                  'scripts',
+                                  'html_lint.py')
+
+    def call_linter(self, path, should_raise=False):
```

Comment:
Instead of these test calling directly the script (which means no coverage is recorded), I would move the main logic from the lint script to the HtmlLint module and call directly the main function.